src/lib/token.js

```diff
@@ -0,0 +1,6770 @@
+  'wotan',
+  'would',
+  'wound',
+  'wove',
+  'woven',
+  'wow',
+  'wrack',
+  'wrap',
+  'wrath',
+  'wreak',
+  'wreck',
+  'wrest',
+  'wring',
+  'wrist',
+  'writ',
+  'write',
+  'writhe',
+  'wrong',
+  'wrote',
+  'wry',
+  'wuhan',
+  'www',
+  'wwww',
+  'wxy',
+  'wyatt',
+  'wyeth',
+  'wylie',
+  'wyman',
+  'wyner',
+  'wynn',
+  'xenon',
+  'xerox',
+  'xxx',
+  'xxxx',
+  'xylem',
+  'xyz',
+  'yacht',
+  'yah',
+  'yak',
+  'yale',
+  'yalta',
+  'yam',
+  'yamaha',
+  'yang',
+  'yank',
+  'yap',
+  'yaqui',
+  'yard',
+  'yarn',
+  'yates',
+  'yaw',
+  'yawl',
+  'yawn',
+  'yea',
+  'yeah',
+  'year',
+  'yearn',
+  'yeast',
+  'yeasty',
+  'yeats',
+  'yell',
+  'yelp',
+  'yemen',
+  'yen',
+  'yet',
+  'yield',
+  'yin',
+  'yip',
+  'ymca',
+  'yodel',
+  'yoder',
+  'yoga',
+  'yogi',
+  'yoke',
+  'yokel',
+  'yolk',
+  'yon',
+  'yond',
+  'yore',
+  'york',
+  'yost',
+  'you',
+  'young',
+  'your',
+  'youth',
+  'yow',
+  'yucca',
+  'yuck',
+  'yuh',
+  'yuki',
+  'yukon',
+  'yule',
+  'yves',
+  'ywca',
+  'yyy',
+  'yyyy',
+  'zag',
+  'zaire',
+  'zan',
+  'zap',
+  'zazen',
+  'zeal',
+  'zealot',
+  'zebra',
+  'zeiss',
+  'zen',
+  'zero',
+  'zest',
+  'zesty',
+  'zeta',
+  'zeus',
+  'zig',
+  'zilch',
+  'zinc',
+  'zing',
+  'zion',
+  'zip',
+  'zloty',
+  'zoe',
+  'zomba',
+  'zone',
+  'zoo',
+  'zoom',
+  'zorn',
+  'zurich',
+  'zzz',
+  'zzzz',
+];
+
+// the below is heavily based on https://andrew.hedges.name/experiments/diceware/
+
+// from: https://www.rempe.us/diceware/
+function secureRandom(count) {
+  var cryptoObj = window.crypto || window.msCrypto;
+  var rand = new Uint32Array(1);
+  var skip = 0x7fffffff - (0x7fffffff % count);
+  var result;
+
+  if (((count - 1) & count) === 0) {
+    cryptoObj.getRandomValues(rand);
+    return rand[0] & (count - 1);
+  }
+
+  do {
+    cryptoObj.getRandomValues(rand);
+    result = rand[0] & 0x7fffffff;
+  } while (result >= skip);
+
+  return result % count;
+}
+
+// based on: http://stackoverflow.com/a/1527820/11577
+function getRandomInt(min, max) {
+  if (window.crypto || window.msCrypto) {
+    return secureRandom(max) + min;
+  } else {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+}
+
+function getToken() {
+  var phrase = [];
+
+  for (var i = 0; i < 3; i += 1) {
+    phrase.push(words[getRandomInt(0, words.length)]);
+  }
+
+  return phrase.join(' ');
+}
+
+// added setLocalToken, hasLocalToken, getLocalToken
+
+// Checks the local storage to see if it has a property with the key 'token' returns string value else null
+export function getLocalToken() {
+  return window.localStorage.getItem('token');
+}
+
+// Checks to see if 'token' exists and returns boolean
+export function hasLocalToken() {
+  return !!getLocalToken();
+}
+
+// Sets token in local storage
+export function setLocalToken() {
+  const token = getToken();
+  const myStorage = window.localStorage;
+  myStorage.setItem('token', token);
+}
```

Comment:
I like these 3 functions, I think they are really handy, nice work! 👏 

This is fine, but it would be better if you move these 3 functions out of this file, as I think that the `lib/token.js` file should take care about just creating the token, and nothing else. And, you can have another file that contains these 3 functions here.

I'm suggesting this because, as it is right now, the way you generate token is tightly coupled with the way you access it.

I envision something like this

```js
// tokenFunctions
import { getToken }  from './tokenGenerator';

export function getLocalToken(...) {...}
export function hasLocalToken(...) {...}
export function setLocalToken(...) {...}
```

In that example, the 3 methods don't know where the token is coming from, and that's good, because that means that in the future you can replace `import { getToken } from './tokenGenerator'` with anything else:
- An API call that gives you the token.
- A DB call that gives you the token.
- Even other token generator file.

Happy to expand this idea beyond if you want, just hit me up in Slack!

Reply:
Good call! Like this breakdown and the explanation 👌🏽

Reply:
Done! This made a lot of sense. 
Thank you!